### PR TITLE
Midterm

### PR DESCRIPTION
**Pull request by mviruet2**
I tried to run it more than 10 times and it did not worked.

### CODE REVIEW

File: q1.py [hunk 1]
```diff
@@ -5,4 +5,8 @@
 #   Example 4: if your balance is 8, then your program should return 8. 
 
 def computeMinimumPayment( balance ):
-    #TODO write code inside this function that achieves the functionality described above
\ No newline at end of file
+    #TODO write code inside this function that achieves the functionality described above
+	if balance < 10:
+		return  balance
+	elif balance < 10:
```
Comment (halfaro1) [inline q1.py]: This `if` and `elif` statement test the same condition.

File: q2.py [hunk 2]
```diff
@@ -26,10 +26,18 @@ def askUserChoice():
     # 1. Asks the user to enter their input (between 1 and 4)
     # 2. Checks that the user's input is valid. If it's not valid (if it's not between 1 and 4), then ask the user to re-enter their input.
     # 3. Once the user enters a valid input, return that input as an integer.
-
+	sticksPicked = int(input('Chose between 1 and 4 sticks'))
+	if 1 <= sticksPicked < 5:
```
Comment (halfaro1) [inline q2.py]: @mviruet2 does this run on your computer? check for errors from the Python IDLE

Comment (mviruet2) [inline q2.py]: @halfaro1 it does not let me run it. I'm going to delete it all and start all over.

File: q2.py [hunk 3]
```diff
@@ -26,10 +26,18 @@ def askUserChoice():
...
+		return sticksPicked
+		
 
 def subtractSticks( number ):
-    global sticks 
+    global sticks
+	sticks -= number
+	if sticks == 1:
```
Comment (halfaro1) [inline q2.py]: @mviruet2 check your indentations here

Comment (mviruet2) [inline q2.py]: @halfaro1 I am having problems with github. Its not letting me commit changes.

**Comment by halfaro1** [comment]
@mviruet2 this is the error I'm getting when I run your program, you should be getting the same error on your Python IDLE:
```
E     File "/home/travis/build/valencia-cop1000-201620/midterm/q2.py", line 34
E       return int (num):
E                       ^
E   SyntaxError: invalid syntax
```

**Comment by mviruet2** [comment]
._. learned that an 's' can make a huge diference @halfaro1

**Comment by halfaro1** [comment]
@mviruet2 
Fantastic work :+1:
I'm marking this pull request as closed, and labeling it as `graded` while I finish grading these projects. When I'm done grading all the projects, and your grade should be on Blackboard, I'll change the tag to `grade updated on blackboard`

**Pull request closed without merging**